en/flight_modes/offboard.md

```diff
@@ -38,11 +38,15 @@ The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL
       > **Note** Acceleration setpoint values are mapped to create a normalized thrust setpoint (i.e. acceleration setpoints are not "properly" supported).
     * Position setpoint **and** velocity setpoint (the velocity setpoint is used as feedforward; it is added to the output of the position controller and the result is used as the input to the velocity controller).
   - PX4 supports the coordinate frames (`coordinate_frame` field): [MAV_FRAME_LOCAL_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_NED) and [MAV_FRAME_BODY_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_BODY_NED).
-* [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET) - Control vehicle attitude/orientation.
+* [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET)

```

Comment:
Thanks @Jaeyoung-Lim - So just to confirm what this means -

* Attitude/orientation with thrust setpoint - this means you can set `SET_ATTITUDE_TARGET.thrust` and `SET_ATTITUDE_TARGET.q`
* Body rate  with thrust setpoint - means you can set `body_roll_rate` ,`body_pitch_rate`, `body_yaw_rate` and `thrust` at the same time?

Reply:
Yes, exactly